CIP-????/README.md

```diff
@@ -0,0 +1,82 @@
+<!-- A short (\~200 word) description of the proposed solution and the technical issue being addressed. -->
+
+## Motivation: why is this CIP necessary?
+<!-- A clear explanation that introduces the reason for a proposal, its use cases and stakeholders. If the CIP changes an established design then it must outline design issues that motivate a rework. For complex proposals, authors must write a Cardano Problem Statement (CPS) as defined in CIP-9999 and link to it as the `Motivation`. -->
+Many of the currently available UPLC builtin functions need to be forced between 1 and 3 times to get rid of type instantiations checked at a higher level language
+of the toolstack (PLC), which most third party tools do not use.
+Moreover, the forces do nothing more than burn cycles of nodes that evaluate contracts, since there is no actual type instantiation happening internally.
+There is always demand to improve performance and reduce resource costs, which is achieved by this proposal.
+
+## Specification
+<!-- The technical specification should describe the proposed improvement in sufficient technical detail. In particular, it should provide enough information that an implementation can be performed solely on the basis of the design in the CIP. This is necessary to facilitate multiple, interoperable implementations. -->
+For all existing UPLC Builtin Functions _x_ that require _n > 0_ forces for evaluation, this proposal suggests to implement the builtin function _x_
+without any required forces.
+
+There are two options for the implementation of this proposal.
+Either the new functions are added to the set of available builtin functions or they replace the current functions.
+This is up to discussion and shifts additional work to either the implementation of UPLC or the implementation of PLC.
+
+Addition:
+ - UPLC needs to support a more diverse set of operations, implying more resources needed for maintainance and secondary implementations
+
+Replacement:
+ - PLC will need to wrap builtin functions with delays internally to emulate the previous behaviour of the builtin functions
+
+
+## Rationale: how does this CIP achieve its goals?
+<!-- The rationale fleshes out the specification by describing what motivated the design and what led to particular design decisions. It should describe alternate designs considered and related work. The rationale should provide evidence of consensus within the community and discuss significant objections or concerns raised during the discussion.
+
+It must also explain how the proposal affects the backward compatibility of existing solutions when applicable. If the proposal responds to a CPS, the 'Rationale' section should explain how it addresses the CPS, and answer any questions that the CPS poses for potential solutions.
+-->
+This proposal reduces the resources needed to evaluate builtin functions by removing the need to apply no-op force operations to them.
```

Comment:
I would really like an impact assessment of this. My suspicion is that the actual performance impact will be negligible, and the main impact will be on making it easier for compiler writers and simplifying the language. As a compiler writer and a maintainer of the language, I appreciate those things, but they're much weaker reasons than widespread performance improvements IMO.